pages/api/replicache-push.ts

```diff
@@ -107,18 +108,20 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
 
   console.log("Processed all mutations in", Date.now() - t0);
 
-  const pusher = new Pusher({
-    appId: "1157097",
-    key: "d9088b47d2371d532c4c",
-    secret: "64204dab73c42e17afc3",
-    cluster: "us3",
-    useTLS: true,
+  const t2 = Date.now();
+
+  const ably = new Ably.Realtime("au5tJQ.cM-6HQ:Bh6Uw0EmAkKqIMaL");
+  const channel = ably.channels.get(`room/${docID}`);
```

Comment:
I wonder if it makes sense to move this out of this function or at least start it earlier. It doesn't look like it is hiding anything async in here but who knows?